src/app/data-entries/dropdown/dropdown-sample-1/dropdown-sample-1.component.ts

```diff
@@ -1,42 +1,14 @@
-import { Component, OnInit, ViewChild } from "@angular/core";
-import {
-    CloseScrollStrategy,
-    ConnectedPositioningStrategy,
-    HorizontalAlignment,
-    IgxDropDownComponent,
-    VerticalAlignment
-} from "igniteui-angular";
+import { Component } from "@angular/core";
 @Component({
     // tslint:disable-next-line:component-selector
     selector: "app-dropdown-sample-1",
     styleUrls: ["./dropdown-sample-1.component.scss"],
     templateUrl: "./dropdown-sample-1.component.html"
 })
-export class DropDownSample1Component implements OnInit {
-    @ViewChild(IgxDropDownComponent) public igxDropDown: IgxDropDownComponent;
-
-    public items: any[] = [];
-
-    private _positionSettings = {
-        horizontalStartPoint: HorizontalAlignment.Left,
-        verticalStartPoint: VerticalAlignment.Bottom
-    };
-    private _overlaySettings = {
-      closeOnOutsideClick: true,
-      modal: false,
-      positionStrategy: new ConnectedPositioningStrategy(this._positionSettings),
-      scrollStrategy: new CloseScrollStrategy()
```

Comment:
We had here Close scroll strategy and in the new sample it will be Absolute scroll strategy!

Reply:
Thanks for pointing this out! I don't think the scroll strategy is really relevant here though, no need to clutter up the example with custom settings.